insights/client/insights_spec.py

```diff
@@ -18,32 +18,17 @@ class InsightsSpec(object):
     '''
     A spec loaded from the uploader.json
     '''
-    def __init__(self, config, spec, exclude):
+    def __init__(self, config, spec):
         self.config = config
-
-        # exclusions patterns for this spec
-        # if exclude is an array of strings, it's old style
-        # if it's an object or an array of dicts, it's new style
-        # use regex if it's defined
-        self.regex = False
-        self.exclude = None
-        if exclude and isinstance(exclude, dict):
-            if 'regex' in exclude and exclude['regex']:
-                logger.debug('Using regular expression matching in remove.conf.')
-                self.regex = True
-                self.exclude = exclude['regex']
-        else:
-            self.exclude = exclude
-        # pattern for spec collection
         self.pattern = spec['pattern'] if spec['pattern'] else None
 
 
 class InsightsCommand(InsightsSpec):
     '''
     A command spec
     '''
-    def __init__(self, config, spec, exclude, mountpoint):
-        InsightsSpec.__init__(self, config, spec, exclude)
+    def __init__(self, config, spec, mountpoint):
+        InsightsSpec.__init__(self, config, spec)
```

Comment:
Any idea why this isn't using `super`?

Reply:
Written many moons ago when I was younger and not as smart. I'll update it since I'm cleaning house anyway